sdk/arm/compute/2019-12-01/armcompute/go.sum

```diff
@@ -1,6 +1,10 @@
 github.com/Azure/azure-sdk-for-go/sdk/azcore v0.8.0 h1:ogZVkNX3wS6ekWS3hD9sQCghWIGfw8CQ9JKv0rQQlVw=
 github.com/Azure/azure-sdk-for-go/sdk/azcore v0.8.0/go.mod h1:UKq2za3CMGx75vfPM9tPSuTBNODR4hX1qAeb+GRoDkc=
+github.com/Azure/azure-sdk-for-go/sdk/azcore v0.8.2 h1:A+8ZU8LjOUX3Ky0mAhY4ct7dWSHjBfa/srPU+M3eLEU=
+github.com/Azure/azure-sdk-for-go/sdk/azcore v0.8.2/go.mod h1:pjLfCjJh+VPLwZylRHc9ZDwRMqHLvfUGH52c/hmBvLs=
```

Comment:
Is this how we expect to output this?

Reply:
The contents of `go.sum` are created by the `go` tooling so this is expected (and quite possibly could change in the future).